controller/src/main/java/io/pravega/controller/util/Config.java

```diff
@@ -139,7 +139,7 @@
     private static final Property<String> PROPERTY_TLS_TRUST_STORE = Property.named("auth.tlsTrustStore", "");
     private static final Property<String> PROPERTY_TLS_KEY_FILE = Property.named("auth.tlsKeyFile", "");
     private static final Property<String> PROPERTY_TOKEN_SIGNING_KEY = Property.named("auth.tokenSigningKey", "");
-    private static final Property<String> PROPERTY_ZK_URL = Property.named("zk.url", "localhost:2121");
+    private static final Property<String> PROPERTY_ZK_URL = Property.named("zk.url", "localhost:2181");
```

Comment:
Unrelated but necessary to change, I also found this problematic in a local deployment.